metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_schema.py

```diff
@@ -150,12 +150,22 @@ def get_query_result(self, query: str) -> RowIterator:
     def get_projects(self) -> List[BigqueryProject]:
         with self.report.list_projects:
             try:
-                projects = self.bq_client.list_projects()
-
-                return [
-                    BigqueryProject(id=p.project_id, name=p.friendly_name)
-                    for p in projects
-                ]
+                projects: List[BigqueryProject] = []
```

Comment:
add a comment explaining why this was required

Reply:
Yes, Let me add the comment.
The thing is we are not able to regenerate the error but from [doc](https://cloud.google.com/bigquery/quotas#api_request_quotas) we got to know two project.lists request is allowed per second and assumed that bigquery client method list_projects is not adding any limit in calling request.
Hence we are trying to add limit in requests per second externally,